lib/resolvers/github.js

```diff
@@ -1,22 +1,20 @@
-const EventEmitter = require('events');
-const { getZelFile } = require('./repository');
+const BaseResolver = require('./base');
+const GitHubFetcher = require('../fetchers/github');
 
-module.exports = class Resolver extends EventEmitter {
-    constructor(options) {
-        super();
-        this.valid = [];
-        this.invalid = [];
-        this.opts = options || {};
+module.exports = class GithubResolver extends BaseResolver {
+    constructor(opts) {
+        super(opts);
+        this.fetcher = new GitHubFetcher();
```

Comment:
I don't think there's any need to assign this _into_ the `GithubResolver`? Declaring above the class may be best, unless you plan on accessing it from outside the Resolver later?

```js
const Resolver = new GitHubResolver();
console.log(Resolver.fetcher);
```